rcl_logging_interface/include/rcl_logging_interface/rcl_logging_interface.h

```diff
@@ -91,6 +98,73 @@ RCL_LOGGING_INTERFACE_PUBLIC
 RCUTILS_WARN_UNUSED
 rcl_logging_ret_t rcl_logging_external_set_logger_level(const char * name, int level);
 
+/// Get the logging directory.
+/**
+ * Uses various environment variables to construct a logging directory path.
+ *
+ * Use $ROS2_LOG_DIR if ROS2_LOG_DIR is set and not empty.
```

Comment:
I don't think we should use `ROS2_` here. The environement variables we already have use just `ROS_`, e.g. `ROS_DOMAIN_ID`.

Reply:
good point. Switched to `ROS_*` in 18b84486e8565a7ab3a9ef319a76f52f0b1906b4 (and will do the same in my ros2/launch PR).